libraries/AP_HAL_ChibiOS/Storage.cpp

```diff
@@ -442,6 +442,12 @@ bool Storage::_flash_erase_ok(void)
  */
 bool Storage::healthy(void)
 {
+#ifdef USE_POSIX
+    // SD card storage is really slow
+    if (_initialisedType == StorageBackend::SDCard) {
```

Comment:
likely could make this better by increasing CH_STORAGE_LINE_SHIFT, try a value of 9 for 512 byte sectors

Reply:
Changed